crates/host/src/module/wasmer_sys.rs

```diff
@@ -118,4 +168,56 @@ mod tests {
             assert_eq!(*deserialized_cached_module, *module);
         }
     }
+
+    #[test]
+    fn cache_get_from_fs() {
```

Comment:
test that:
- builds module, serializes, writes to filesystem at expected path
- gets module from ModuleCache
- asserts that module now exists in deserialized in-memory path